src/js/libcs/Accessors.js

```diff
@@ -219,6 +221,11 @@ Accessor.setField = function(geo, field, value) {
             geo.pinned = value.value;
         }
     }
+    if (field === "labeled" || field === "labelled") {
```

Comment:
If i'm reading the code correctly this only applies to the getter code. We never defined something like `if (Accessor.generalFields[field])` for the setter function which could use the mapping.